math/CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ Ref: https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.j
 
 ### Improvements
 
+* [#17441](https://github.com/cosmos/cosmos-sdk/pull/17441) feat(math): Dec alias from LegacyDec
```

Comment:
nit, can we drop that `feat(math):` and make a sentence of it.

Reply:
Done